src/main/controllers/service/ServicePostcodeResultsController.ts

```diff
@@ -13,11 +13,41 @@ export class ServicePostcodeResultsController {
     private readonly api: FactApi
   ) { }
 
+  public async getCourtResultsByPostcode(req: FactRequest, res: Response): Promise<void> {
+    const postcode  = req.query.postcode? (req.query.postcode as string).toLowerCase() : '';
```

Comment:
Super minor issue....missing space between 'postcode' and '?'. Not sure typescript but Java checkstyle will compliant about this.